apirouter.go

```diff
@@ -102,7 +102,9 @@ func newAPIRouter(app *appContext, userRealIP bool) apiMux {
 		})
 		r.Route("/pool", func(rd chi.Router) {
 			rd.With(app.BlockIndexLatestCtx).Get("/", app.getTicketPoolInfo)
+			rd.With(app.BlockIndexLatestCtx).Get("/full", app.getTicketPool)
 			rd.With(BlockIndexPathCtx).Get("/b/{idx}", app.getTicketPoolInfo)
+			rd.With(BlockIndexOrHashPathCtx).Get("/b/{idxorhash}/full", app.getTicketPool)
```

Comment:
b?  SHouldn't we have something more descriptive?

Reply:
perhaps.  We were using `b` for block and `r` for range to keep the path length under control